Add is_csv kwarg to prepare_upload util (#165)
alphagov/notifications-python-client: DOCUMENTATION.md

    }
```

#### CSV Files

> Uploads for CSV files should be handled a little differently - using the `is_csv` parameter

Reviewer: ```suggestion
Uploads for CSV files should use the `is_csv` parameter
```
